### PR TITLE
Set default answers

### PR DESCRIPTION
**Pull request by nikolashkubelev**
- [x] Set NumberOfLocations to 1 by default

### CODE REVIEW

File: src/components/Questionnaire.vue [hunk 1]
```diff
@@ -95,7 +95,7 @@ export default defineComponent({
       this.applicationId = preApplicationRes.data.id;
       this.questions = preApplicationRes.data.questionnaire.questions;
       this.sections = preApplicationRes.data.questionnaire.layout;
-      this.answers = {};
+      this.answers = { NumberOfLocations: 1 };
```
Comment (rthreei) [inline src/components/Questionnaire.vue:98]: Why is this necessary?

Comment (GigaGyrfalcon) [reply src/components/Questionnaire.vue:98]: We decided to include NumberOfLocations to pre-render address questions for a pre-application example.

Comment (rthreei) [reply src/components/Questionnaire.vue:98]: That shouldn't be necessary though.  We should not expect the host/developer to set this value.  This should be a concern for the component.

Comment (GigaGyrfalcon) [reply src/components/Questionnaire.vue:98]: I agree with you. We already have an issue: https://github.com/briza-insurance/briza-air/issues/743
This is a temporary solution to avoid confusion with passed questions and rendered questions for developers who will clone and use this repo.

Comment (rthreei) [reply src/components/Questionnaire.vue:98]: I don't want us to confuse developers with `this.answers = { NumberOfLocations: 1 };`

We should prioritize https://github.com/briza-insurance/briza-air/issues/743 as an urgent asap hotfix.

Comment (nikolashkubelev) [reply src/components/Questionnaire.vue:98]: https://github.com/briza-insurance/briza-air/pull/830/files
